executor/executor_distsql.go

```diff
@@ -294,7 +302,28 @@ func closeAll(objs ...Closeable) error {
 	return nil
 }
 
-// XSelectIndexExec represents XAPI select index executor without result fields.
+// XSelectIndexExec represents DistSQL select index executor.
+// There are two execution modes. one is single-read, in which case we only need to read index keys.
```

Comment:
one is single-read, in which case we only need to read index keys.
-> One is single-read, in which case we only need to read the index keys.